NOMENCLATURE.md

```diff
@@ -16,102 +16,102 @@ To begin, we'll define the concept of a _cluster_.
 
 ### Cluster
 
-A _Cluster_ is made up of an organized set of Docker engines configured in a
+A _cluster_ is made up of an organized set of Docker _Engines_ configured in a
 manner to allow the dispatch of _services_.
 
 ### Node
 
-A _Node_ refers to an active member in a cluster. Nodes can execute work and
-act as a cluster manager.
+A _Node_ refers to an active member in a cluster. Nodes can execute work and / or
+act as a cluster _manager_.
 
 ### Manager
 
-A manager accepts _services_ defined by users through the Cluster API. When a
+A _manager_ accepts _services_ defined by users through the cluster API. When a
 valid _service_ is provided, the manager will generate tasks, allocate resources
 and dispatch _tasks_ to an available _node_.
 
-_Managers_ operate in a coordinated group, organized via the Raft protocol.
-When quorum is available, a leader will be elected to handle all API requests.
+_managers_ operate in a coordinated group, organized via the Raft protocol.
+When a quorum is available, a leader will be elected to handle all API requests
+and all other members of the quorum will proxy to the leader.
 
 #### Orchestrator
 
 The _Orchestrator_ ensures that services have the appropriate set of tasks
-running in the cluster, according the service configuration and polices.
+running in the _cluster_ according to the _service_ configuration and polices.
```

Comment:
Do we really need so many italics? 😃

For example, we've already introduced and explained the term `cluster`, so I don't think it needs to be italicized in subsequent uses in the document.